### ISSUE TITLE
NVDA Hides Elements on a Webpage

### ISSUE THREAD

**Issue by frastlin** [nvaccess/nvda#10800]
### Steps to reproduce:

1. Visit [Quickbooks Online](https://qbo.intuit.com/redir/testdrive)
2. Hit ctrl+home and arrow down until you hear "Heading level 2 Sample Company"

### Actual behavior:

NVDA does not read the left navigation panel.

### Expected behavior:

NVDA will read the left navigation panel.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2019.3

#### Windows version:

Windows 10 64 bit

#### Name and version of other software in use when reproducing the issue:

Both:
Chrome Version 80.0.3987.106 (Official Build) (64-bit)
Firefox 73.0 (64-bit)

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

This just started occurring when I updated to NVDA 2019.3. I never had any problems with NVDA 2019.2.1.

### Other Comments

If you select all with NVDA running and copy and paste the content into another document, the nav bar will not be there. If you turn off NVDA, and copy and paste into a text file, the nav bar will be there.
If you hit shift-tab  on the "Expand/Collapse Navigation" button, then you will hear the links on the side bar, but when you hit enter, you will only click on the "Expand/Collapse Navigation" button.

**Comment by Adriani90** [comment]
Some things to check:
1. in NVDA settings / browse mode, is "preserve screen layout if supported" enabled or disabled?
2. In NVDA / advanced settings, is "automatically move system focus to focusable elements" enabled or disabled?

Try to reproduce this issue with both options disabled.

**Comment by frastlin** [comment]
I disabled "automatically move system focus to focusable elements" and was still able to reproduce the issue.
I always have the "Preserve screen layout" disabled.

**Comment by Adriani90** [comment]
cc: @jcsteh maybe this has something with the recent PR introduced by you? However, I am not sure. I see there a lot of element in indexInParent 0. Could this be the reason?
After testing futher it seems NVDA really does not recognize the elements in browse mode, but it reports them when navigating with tab in focus mode.

**Comment by frastlin** [comment]
The Intuit team believes that in the latest updates to Firefox and Chrome, aria-owns began to be supported, and there was a mis-use of aria-owns in the sidebar that should have been aria-controls.

**Comment by jcsteh** [comment]
> The Intuit team believes that in the latest updates to Firefox and Chrome, aria-owns began to be supported, and there was a mis-use of aria-owns in the sidebar that should have been aria-controls.

They're correct about the incorrect usage of aria-owns being the problem. However, aria-owns has been supported in Firefox and Chrome for quite some time now, so it's not a change in recent versions of browsers.

**Comment by jcsteh** [comment]
Closing, since this is clearly authoring error.
